mantle/cmd/kola/devshell.go

```diff
@@ -95,22 +95,13 @@ func runDevShellSSH(builder *platform.QemuBuilder, conf *v3types.Config) error {
 		return fmt.Errorf("stdin is not a tty")
 	}
 
-	tmpd, err := ioutil.TempDir("", "kola-devshell")
+	sshPubKeyBuf, tmpd, sshKeyPath, err := util.CreateSSHAuthorizedKey()
```

Comment:
Optional: It seems weird to me to have this call create a temporary directory.  In this case it just happens to be the first thing we write into the tempdir is a SSH key, but in e.g. the testiso cases I think we already have other temporary directories and we could put they key inside those.

But anyways not a blocker, just optional.